easybuild/easyconfigs/p/ParaView/ParaView-5.2.0_python-six-update.patch

```diff
@@ -0,0 +1,17 @@
+Update for VTK third party dependency PythonSix(https://pypi.python.org/pypi/six)
+From version 1.7.2 to using System Python one
+
+This update is necessary because ParaView uses his version of six.py(v1.7.2) instead of the Python one, matplotlib plugin for Paraview needs six.py(>v.1.10.0).
+
+--- a/VTK/ThirdParty/AutobahnPython/module.cmake        2016-11-14 16:41:57.000000000 +0100
++++ b/VTK/ThirdParty/AutobahnPython/module.cmake        2017-11-21 09:16:40.088867103 +0100
+
+@@ -1,6 +1,5 @@
+ vtk_module(AutobahnPython
+   DEPENDS
+-    SixPython
```

Comment:
Hmm, where is it specified that `six` is `1.7.2`?

Reply:
in source ParaView VTK/ThirdParty/SixPython/six.py